src/components/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -94,6 +112,13 @@ export const MarkdownEditor = () => {
     }
   };
 
+  const onChange = (editorState: EditorState) => {
+    editorState.read(() => {
+      const markdown = $convertToMarkdownString(PLAYGROUND_TRANSFORMERS);
+      setContentWritten(markdown);
+    });
```

Comment:
Denne kommer vel bare til å bli tyngre og tyngre for hvert tastatrykk, tror du ikke? Kanskje vi heller skulle funnet ut av hvordan man kan kun trenger å konvertere til markdown når man faktisk vil lagre? Kan f.eks sende inn en custom lagre-knapp hvis du er bekymret for gjenbrukbarhet!

Reply:
Den blir marginalt tyngre for 300 characters, men jeg kan sjekke det ut! 👍🏿